metals/src/main/scala/scala/meta/internal/metals/InitializationOptions.scala

```diff
@@ -0,0 +1,37 @@
+package scala.meta.internal.metals
+import com.google.gson.JsonElement
+import org.eclipse.{lsp4j => l}
+
+final case class InitializationOptions(
+    statusBar: String = "off",
```

Comment:
I'd recommend trying to stay as close to the names we use in `ClientExperimentalCapabilities`.

https://github.com/scalameta/metals/blob/8dc77885d48f5ebd7b97f59b17c6ef850beb7c73/metals/src/main/scala/scala/meta/internal/metals/ClientExperimentalCapabilities.scala#L6-L16